.travis.yml

```diff
@@ -13,6 +13,10 @@ services:
 install:
   - make install
 
+before_script:
+  - make restore_schema $TEST_WAREHOUSE_URL ./tests/dbs/warehouse_schema_dump.sql True
```

Comment:
@vitorbaptista  As I understand it, this should retrieve the `TEST_WAREHOUSE_URL` from the environment  and run this command after which `make test` should pass. These variables should be set up just like the other database URLs. 
I tried to test this locally in a docker image that simulates the Travis build but I can't figure how to properly install the requirements.
Please give me a heads up after you add  `TEST_WAREHOUSE_URL` and `TEST_DATABASE_URL` to the repo settings.